config/externalname.go

```diff
@@ -1724,6 +1724,11 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	//
 	// Media Convert Queue can be imported via the queue name
 	"aws_media_convert_queue": config.NameAsIdentifier,
+
+	// emr
+	//
+	// EMR Security Configurations can be imported using the name
+	"aws_emr_security_configuration": config.NameAsIdentifier,
```

Comment:
Could you please remove this resource from the externalnamenottested.go file?

Reply:
Done, thank you!